papaslib/CMakeLists.txt

```diff
@@ -1,24 +1,11 @@
 cmake_minimum_required(VERSION 2.8)
 
 # use, i.e. don't skip the full RPATH for the build tree
-SET(CMAKE_SKIP_BUILD_RPATH  FALSE)
+SET(CMAKE_SKIP_BUILD_RPATH  TRUE)
```

Comment:
Can you tell me more about what is being done with RPATH? 
Are we supposed to make it when we're on macos but not on lxplus? 
If yes, couldn't this be decided based on the architecture instead of hardcoded?